Sources/Purchasing/Purchases/PurchasesType.swift

```diff
@@ -1233,6 +1258,24 @@ public protocol PurchasesSwiftType: AnyObject {
     func eligibleWinBackOffers(
         forPackage package: Package
     ) async throws -> [WinBackOffer]
+
+    /**
+     * Fetches the virtual currencies for the current subscriber.
+     *
+     * - Returns: The ``VirtualCurrencies`` object containing the virtual currencies for the subscriber.
+     *
+     * #### Related Articles
+     * -  [Virtual Currencies](https://www.revenuecat.com/docs/offerings/virtual-currency)
+     */
+    func virtualCurrencies() async throws -> VirtualCurrencies
+
+    /**
+     * Invalidates the cache for virtual currencies.
+     *
+     * This is useful for cases where a virtual currency's balance might have been updated
+     * outside of the app, like if virtual currency is granted through the RevenueCat dashboard.
```

Comment:
I think this is example might be a bit of a less common use case. The typical scenario where you'd use this method is when the balance has changed because you have either decreased it (user spending) or increased it (new grant) from your backend using server APIs.